### PR TITLE
#864: Remove ksonnet from AWS Azure deployment pages

### PR DESCRIPTION
**Pull request by scottilee**
- Removed references to ksonnet on AWS deployment page
- Removed references to ksonnet on Azure deployment page
- Removed unnecessary whitespace

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/911)

### CODE REVIEW

File: content/docs/aws/deploy/install-kubeflow.md [hunk 1]
```diff
@@ -51,9 +50,6 @@ Your Kubeflow `app` directory contains the following files and directories:
     * This directory is created when you run `kfctl.sh generate platform`.
     * You can modify the `cluster_config.yaml` and `cluster_features.sh` files to customize your AWS infrastructure.
 * **${KFAPP}/k8s_specs** - A directory that contains YAML specifications for daemons deployed on your Kubernetes Engine cluster.
-* **${KFAPP}/ks_app** - A directory that contains the [ksonnet](https://ksonnet.io/) application for Kubeflow.
```
Comment (richardsliu) [inline content/docs/aws/deploy/install-kubeflow.md]: Can you add one line here about the kustomize file directory? See https://github.com/kubeflow/website/pull/887/files

File: content/docs/azure/deploy/install-kubeflow.md [hunk 2]
```diff
@@ -43,10 +40,6 @@ Your Kubeflow `app` directory contains the following files and directories:
 * **app.yaml** - Defines the configuration related to your Kubeflow deployment.
     * These values are set when you run `kfctl init`.
     * These values are snapshotted inside `app.yaml` to make your app self contained.
-* **${KFAPP}/ks_app** - A directory that contains the [ksonnet](https://ksonnet.io/) application for Kubeflow.
```
Comment (richardsliu) [inline content/docs/azure/deploy/install-kubeflow.md]: Same here

**Comment by scottilee** [comment]
@richardsliu I think I did what you wanted.

**Pull request merged**